app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CommentsController < ApplicationController
+  before_action :find_post
+  before_action :find_comment, only: [:edit, :update, :destroy]
+
+  def create
+    @comment = @post.comments.new comment_params
```

Comment:
I know we send params to function without `()`. But we should do it because it's easier to read. `@comment = @post.comments.new(comment_params)`